src/status_im/multiaccounts/update/core.cljs

```diff
@@ -1,17 +1,30 @@
 (ns status-im.multiaccounts.update.core
-  (:require [status-im2.constants :as constants]
-            [utils.re-frame :as rf]
-            [status-im.utils.types :as types]
-            [taoensso.timbre :as log]))
+  (:require [status-im.utils.types :as types]
+            [status-im2.constants :as constants]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
 
-(rf/defn send-multiaccount-update
+(rf/defn send-contact-update
   [{:keys [db] :as cofx}]
-  (let [multiaccount                          (:multiaccount db)
-        {:keys [name preferred-name address]} multiaccount]
+  (let [multiaccount                                       (:multiaccount db)
```

Comment:
Looks like the `multiaccount` binding is unnecessary, since all you want is to destructure the map.